packages/sveltekit/src/vite/autoInstrument.ts

```diff
@@ -89,24 +91,57 @@ export function makeAutoInstrumentationPlugin(options: AutoInstrumentPluginOptio
  */
 export async function canWrapLoad(id: string, debug: boolean): Promise<boolean> {
   const code = (await fs.promises.readFile(id, 'utf8')).toString();
+  const mod = parseModule(code);
 
-  const codeWithoutComments = code.replace(/(\/\/.*| ?\/\*[^]*?\*\/)(,?)$/gm, '');
-
-  const hasSentryContent = codeWithoutComments.includes('@sentry/sveltekit');
+  const hasSentryContent = mod.imports.$items.find(importExpression => importExpression.from === '@sentry/sveltekit');
```

Comment:
does this skip if I import sentry to use top level api but not wrap load?

```ts
import * as Sentry from '@sentry/sveltekit';

export function load({ params }) {
    Sentry.setTag('stuff', 'here');
    return {
        post: {
            title: `Title for ${params.slug} goes here`,
            content: `Content for ${params.slug} goes here`
        }
    };
}
```

Reply:
Yes, but this didn't change from the previous check. For now, we bail if we find an SDK import which we take as the minimum indication that something is instrumented in this file.

Reply:
Yup, just realized that previously this same behavior was there. Could we be more strict and check for `wrapperFunction` existence (like `wrapLoadWithSentry`)?

Reply:
I guess the AST walking would be a little more involved but I think so, yes. The thing is, I'm not sure if this has the potential to collide with custom instrumentation that doesn't use the wrapper. For example, if anyone creates a transaction manually for the load function. But I guess in this scenario, our span/transaction would be thrown away which should be fine 🤔